package.json

```diff
@@ -18,7 +18,8 @@
     "start:dev": "npm run build && concurrently -k -p \"[{name}]\" -n \"Views,TypeScript,Node\" -c \"yellow.bold,cyan.bold,green.bold\" \"npm run watch-views\" \"npm run watch-ts\" \"npm run watch-node\"",
     "start:feature": "set -a; source feature.env; set +a && npm run build && npm start",
     "start-mocks": "node mock-server/start.js",
-    "test": "jest --runInBand --testPathPattern=/test/.*",
+    "test:ci": "jest --runInBand --testPathPattern=/test/.*",
+    "test": "jest --testPathPattern=/test/.*",
```

Comment:
the isolated modules change massively reduces memory usage as well as speeding up tests.
This reduced memory requirements mean that locally we can run the tests in parallel - this combined means the tests run much faster: it knocks over a minute off the test run on my laptop.